dashboard/app/controllers/api/v1/sections_students_controller.rb

```diff
@@ -24,6 +24,26 @@ def update
     end
   end
 
+  # Used only for picture and word sections
+  # POST /sections/<section_id>/students/bulk_add
+  def bulk_add
+    puts "hi we're going to bulk add some students"
+    new_students = []
+    params[:students].each do |student|
+      new_student = User.create(
+        age: student["age"],
+        name: student["name"],
+        gender: student["gender"],
+        sharing_disabled: student["sharing_disabled"],
+        user_type: User::TYPE_STUDENT
+      )
+      @section.add_student(new_student)
```

Comment:
Fails at: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/sections/section.rb#L168

With error:
`Validation failed: Student user user type is not included in the list, Student user age is required, Student user password is required, Student user email is required`